features/webcodecs.yml.dist

```diff
@@ -4,9 +4,9 @@
 status:
   baseline: false
   support:
-    chrome: "106"
-    chrome_android: "106"
-    edge: "106"
+    chrome: "126"
+    chrome_android: "126"
+    edge: "126"
```

Comment:
@foolip do you care about this? I was thinking that we might want to allow all single-engine features to advance like this, unless and until there's some non-overlapping set of keys between browsers. That is to say, the first cross-browser subset would prompt us to freeze the feature's "birthday" in each browser, but until then it's a primordial feature we shouldn't stress over.

Reply:
I think we should use `compute_from` here just as we would for a Baseline feature. If we wait until it's more widely supported until we add it, then we'd end up changing the versions claimed for the overall feature at that point.

Reply:
Done in https://github.com/web-platform-dx/web-features/pull/1296.